### PR TITLE
Cstm   add pet id/UUID   cpc 791

### PR DESCRIPTION
**Pull request by Nmazza1**
JIRA:
https://champlainsaintlambert.atlassian.net/browse/CPC-791?atlOrigin=eyJpIjoiZWU3MTFjNWQ0ZmYwNDAxYjhlODUxNWQyNTI4MDA3ODIiLCJwIjoiaiJ9
## Context: 
This commit adds a public identifier for the Pet Model and a private databases identifier.
## Changes
-Added public petId
-Changed Pet Testing Methods
## Dev notes (Optional)
The tests all pass so there should not be any issues when handling CRUD operations on Pets, if there is come tell me so I can send corresponding fixes.

### CODE REVIEW

**Review by DisgruntledMensch — APPROVED** [review]
looks good

File: customers-service-reactive/src/main/java/com/petclinic/customersservice/customersExceptions/http/HttpErrorInfo.java [hunk 1]
```diff
@@ -25,20 +27,4 @@ public HttpErrorInfo() {
         this.message = null;
     }
 
-    public ZonedDateTime getTimestamp() {
```
Comment (mattjulienn) [inline customers-service-reactive/src/main/java/com/petclinic/customersservice/customersExceptions/http/HttpErrorInfo.java:28]: In all of our projects with Christine, we always have this usually so I'm not sure whether or not this is a 100% prerequisite but definitely something I'd keep for exception handling, but that's just me...

**Review by mattjulienn — CHANGES_REQUESTED** [review]
read the comment, family

File: customers-service-reactive/src/main/java/com/petclinic/customersservice/customersExceptions/http/HttpErrorInfo.java (hunk 1 above)
Comment (Nmazza1) [reply customers-service-reactive/src/main/java/com/petclinic/customersservice/customersExceptions/http/HttpErrorInfo.java:28]:
```
  public ZonedDateTime getTimestamp() {
        return this.timestamp;
    }
```
There is a method that pulls a timestamp

Comment (mattjulienn) [reply customers-service-reactive/src/main/java/com/petclinic/customersservice/customersExceptions/http/HttpErrorInfo.java:28]: looks good then, brother

**Review by mattjulienn — APPROVED** [review]
looks great, family

**Pull request merged**